db/migrate/20170606115344_change_lookup_key_path_to_text.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeLookupKeyPathToText < ActiveRecord::Migration
+  def change
+    remove_index :lookup_keys, :path
```

Comment:
I'm removing the index since indexing text on mysql is problematic and we never search based on the path anyways.